tests/functional/test_libraries.py

```diff
@@ -405,9 +405,15 @@ def test_PyQt5_Qt(pyi_builder):
                             **USE_WINDOWED_KWARG)
 
 
-@xfail(True, reason="Hook is old and needs updating.")
 @importorskip('PySide2')
 def test_PySide2_QWebEngine(pyi_builder, data_dir):
+    if is_darwin:
+        # This tests running the QWebEngine on OS X. To do so, the test must
+        # run only a onedir build -- onefile builds don't work.
```

Comment:
This comment is misleading for me. "This tests … on OS X", but the test runs on all platforms, doesn't it?!

Reply:
Changed to explicitly indicate it's only skipping the onefile OSX build